rskj-core/src/main/java/org/ethereum/rpc/FilterRequest.java

```diff
@@ -78,4 +84,56 @@ public String getBlockHash() {
     public void setBlockHash(String blockHash) {
         this.blockHash = blockHash;
     }
+
+    public boolean isValid() {
+        if (fromBlock != null) {
+            BnTagOrNumberValidator.isValid(fromBlock);
+        }
+        if (toBlock != null) {
+            BnTagOrNumberValidator.isValid(toBlock);
+        }
+        validateAddress();
+        validateTopics();
+        if (blockHash != null) {
+            BlockHashValidator.isValid(blockHash);
+        }
+        return true;
+    }
+
+    private boolean validateTopics() {
```

Comment:
same as in other validators, this method always returns `true`. What about `private void validateTopics()`?